<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCAI</class>
 <widget class="QWidget" name="CCAI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_ccai_sidebar">
     <property name="maximumSize">
      <size>
       <width>160</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>CCAI CHAT</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_ccai_sidebar_newchat">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>新建对话</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_ccai_sidebar_search">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>搜索历史对话</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="listView_ccai_sidebar_chatlist"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_ccai_sidebar_login">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>CCOS</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_ccai_chat">
     <property name="title">
      <string>view</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_ccai_chat_model">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
        <property name="title">
         <string>选择模型</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QComboBox" name="comboBox_ccai_chat_model_select">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_ccai_chat_model">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_ccai_chat_tools_select">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>默认聊天</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>生成文章</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>生成音乐</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>生成视频</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>AI换脸</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>AI换声</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser_ccai_chat_output"/>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_ccai_chat_input">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>120</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="title">
         <string>对话</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="6" colspan="2">
          <widget class="QPushButton" name="pushButton_ccai_chat_input_send">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>发送</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButton_ccai_chat_input_enableweb">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>网页搜索</string>
           </property>
          </widget>
         </item>
         <item row="1" column="6" rowspan="2">
          <widget class="QPushButton" name="pushButton_ccai_chat_input_file">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>上传附件</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" rowspan="2" colspan="4">
          <widget class="QTextEdit" name="textEdit_ccai_chat_input">
           <property name="minimumSize">
            <size>
             <width>680</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="7" rowspan="2">
          <widget class="QPushButton" name="pushButton_ccai_chat_input_img">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>上传图片</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QComboBox" name="comboBox_ccai_chat_input_deep">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_ccai_chat_input_enablethink">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>使用推理</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <spacer name="horizontalSpacer_ccai_chat_input">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
